keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -207,7 +207,7 @@ since users need to explicitly set the annotation.
 None.
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-We will do manual testing.
+Manually tested, worked as expected.
```

Comment:
Line 196: given you wrote that those unit tests already exist - can you please update the answer in that line.

Reply:
ah, sorry, didn't notice it said "we will" :) updated.